components/workouts/Workouts.jsx

```diff
@@ -46,10 +49,21 @@ export default () => {
     items.push({ color: '#00000000', name: '', displayEllipses: false });
   }
 
-  return (
+  const Stack = createStackNavigator();
+  return ( // This allows you to access 2 different pages on same navigation tab. (i.e. Workouts)
```

Comment:
@shriyan44 is this the functionality we wanted here?, also if it is I think we should include a back button too.